src/main.ts

```diff
@@ -192,9 +185,7 @@ function toClosureJS(options: ts.CompilerOptions, fileNames: string[]):
     jsFiles[fileName] = output;
   }
 
-  jsFiles[internalExternsFileName] = tsickleExterns;
```

Comment:
Ah yep, this was to get the externs file into the list of input files to the closure command.